src/mkdocstrings/extension.py

```diff
@@ -78,7 +78,7 @@ def __init__(
         self._config = config
         self._handlers = handlers
         self._autorefs = autorefs
-        self._updated_env = False
+        self._updated_envs: list = []
```

Comment:
I believe a `set` would be more appropriate.

Reply:
Updated.